### ISSUE TITLE
Tinted tiles are unclear

### ISSUE THREAD

**Issue by tomchadwin** [mapswipe/app-feedback#5]
Once tapped, the tint on tiles makes them much less clear. This is problematic if you have to refer to tiles you've already tapped, in order to investigate eg roads etc. Perhaps a thin coloured border might work better?

**Comment by althio** [comment]
I also think coloured overlay are problematic for clear view, and even more so for colour blind people.
The thin border is only slightly better in my opinion.
I would rather suggest a small sign in the middle of the tile, as overlay:
- :white_check_mark: [tick, check mark: OK]
- :white_check_mark: :question: [tick and question mark: maybe]
- :no_entry_sign: [no or bad imagery]

**Comment by tomchadwin** [comment]
I disagree - the symbol could easily obscure an important feature on the map. I take your point about colour
Perhaps solid border, dashed, and dotted could differentiate, along with colour.
